src/phenCryo.ttl

```diff
@@ -77,14 +120,35 @@ sophcr:IceCalving rdf:type owl:Class ;
 
 ###  http://sweetontology.net/phenCryo/Periglacial
 sophcr:Periglacial rdf:type owl:Class ;
-                  rdfs:subClassOf sophcr:GlacialProcess ;
-                  rdfs:label "periglacial"@en .
+                  rdfs:subClassOf sophcr:PhysicalProcess ;
+                  owl:disjointWith sophcr:GlacialProcess ;
+                  rdfs:label "periglacial"@en ;
+                  skos:note "Subclass relation changed from Glacial Process to Physical Process because Periglacial involves frost action not associated with glacial processes"@en ;
+                  rdfs:seeAlso <https://github.com/ESIPFed/sweet/issues/185> ;
+                  skos:definition  [
+                    rdfs:comment  "A frost action process which occurs in cold environments which are not part of a glacier."@en ;
```

Comment:
See #185

Reply:
Collective comments from our [Semantic Harmonization call](
https://docs.google.com/document/d/1IFA6AsCdxESh9mZB5W1bZ_vARFkF3OvnL-KgdGNcp60/edit#heading=h.2uotl4egsd38)

As above, this could be mapped to [ENVO:Periglacial](http://purl.obolibrary.org/obo/ENVO_01001853), however, #185 shows the need to revise the whole branch in SWEET before we make such mappings.

Reply:
see #193 and comment above on Englacial.